src/apps/dialogs/AbilityCheckRollDialog.svelte

```diff
@@ -62,6 +62,7 @@
 
     let visibilityMode = options.visibilityMode ?? game.settings.get("core", "rollMode");
 
+    let { minRoll } = options.minRoll ?? $actor.system.abilities[abilityKey].check;
```

Comment:
Would be better to default to 1 if the ability is not selected for some reason. Also this will fail cz you can't extract `minRoll` from `options.minRoll`.

```ts
let minRoll = options.minRoll ?? $actor.system.abilities[abilityKey]?.check.minRoll ?? 1;
```